source/tests/pt_expt/test_lmdb_training.py

```
@@ -567,6 +733,205 @@ def test_mixed_nloc_statistics_and_training(self) -> None:
+    def _se_e2_a() -> dict:
+        return {
+            "type": "se_e2_a",
+            "sel": [6, 12],
+            "rcut_smth": 0.5,
+            "rcut": 3.0,
+            "neuron": [8, 16],
+            "axis_neuron": 4,
+            "seed": 1,
+        }
+
+    def _run(self, descriptor: dict, *, compile: bool = False):
```

Ruff (0.16.1)

[error] Function argument `compile` is shadowing a Python builtin

(A002)